ocaml/src/albamgr_test.ml

```diff
@@ -20,9 +20,9 @@ open Albamgr_protocol.Protocol
 
 let host = "127.0.0.1"
 let hosts = [ host ]
-let ccfg_ref = ref None
+let ccfg_url_ref = ref None
 
-let get_ccfg () = Option.get_some (!ccfg_ref)
+let get_ccfg_url () = Option.get_some (!ccfg_url_ref)
```

Comment:
The url would've already been resolved to a Arakoon_config.t here, so that's no longer applicable for the code that uses this...
For what you're saying the change of type between alba_cfg_url and alba_cfg_file in cli_common already prevents that.